### DISCUSSION TITLE
Zitadel Self hosted & Cloud, UI Security issue in console

### DISCUSSION THREAD

**Discussion by stiwari99** [zitadel/zitadel#7269] [Q&A]
Recently we deployed self hosted version of zitadel behind nginx proxy and cloudflare in front. Everything was smooth but we are getting below error in browser console 
```
Wait! 
Inserting something here could give attackers access to your zitadel account. 
If you don't know exactly what you're doing, close the window and stay on the safe side 
If you know exactly what you are doing, you should work for us
```

<img width="1145" alt="Screen Shot 2024-01-20 at 11 50 08 PM" src="https://github.com/zitadel/zitadel/assets/585384/462a9677-7d31-4ef0-9fc0-cd357df2b2d3">

We were not able to find anything documented related to this error, could anyone please help

### Zitadel.cloud 
Zitadel.cloud also is also giving similar error in console, so its not just self hosted. Do we know what is this?

**Comment by hifabienne** [comment]
@peintnermax can you please have a look at this?

**Comment by HungryHowies** [comment]
Good Catch @stiwari99  

![image](https://github.com/zitadel/zitadel/assets/22652276/85c0eb5b-0bf2-4a4a-ad8a-3d6bb0df45eb)

**Comment by peintnermax** [comment]
The `wait ...` is not an error but a hardcoded warning for users not to play around in the console to expose their data to attackers. 
Its currently part of the console build and is not configurable.
